src/components/footer/footer.module.scss

```diff
@@ -5,7 +5,7 @@
   display: flex;
   overflow: hidden;
   height: 40vh;
-  background-color: $footer-container;
+  background-color: $footerContainer;
```

Comment:
Please, rename this color variable  to one that is a color as name

Reply:
Resolved in the PR of the base branch.